nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -20,6 +20,15 @@
 
 - [Apache Guacamole](https://guacamole.apache.org/), a cross-platform, clientless remote desktop gateway. Available as [services.guacamole-server](#opt-services.guacamole-server.enable) and [services.guacamole-client](#opt-services.guacamole-client.enable) services.
 
+- Support for WiFi6 (IEEE 802.11ax) and WPA3-SAE-PK was enabled in the `hostapd` package.
```

Comment:
This line should go under "highlights" and the rest either in "backward incompatibilities" or "notable changes".
"New services" seems the wrong section.